crates/astria-sequencer-relayer/tests/blackbox/helper.rs

```diff
@@ -307,13 +307,23 @@ impl TestSequencerRelayer {
     }
 }
 
+// allow: this is not performance-critical, with likely only one instance per test fixture.
+#[allow(clippy::large_enum_variant)]
+pub enum CometBftBlockToMount {
+    GoodAtHeight(u32),
+    BadAtHeight(u32),
+    Block(tendermint::Block),
+}
```

Comment:
just flagging that i updated these tests to use `SequencerBlock`s into of cometbft blocks, will need to be updated

Reply:
Thanks!  Unfortunately a recent PR deleted most of these tests, but #1008 will see them restored.  I'll merge our changes so when the tests are restored, we can reintroduce the test I initially added in this PR, but which has been deleted after merging `main`.

Reply:
Done in [81686cc](https://github.com/astriaorg/astria/pull/1001/commits/81686cc4a3d4e1540b2f594e06e622b6dc597dc8).